### PR TITLE
Update Redirect statergy

### PR DESCRIPTION
**Pull request by sumitshinde-84**
## Description

Currently, we are using a meta refresh for redirects, which is not considered SEO-friendly and negatively impact SEO. Also meta-refersh method does not clearly indicate a redirect to search engine crawlers bot, as they typically recognize only 301 and 302 status codes. Meta refresh redirects return a 200 status code, which does not efficiently transfer SEO equity from the old URL to the new one.

In the current pull request, we have implemented server-side redirects that correctly respond with a 301 status code. However, the hyperlink test on github actions is failing, maybe because it expects a 200 status code.

To check the redirects, use the following tool: 

tool : https://www.redirect-checker.org/
test with meta refresh : `https://flowfuse.com/blog/first-deploy/`
test with server side redirect: `https://deploy-preview-2698--flowforge-website.netlify.app`

For further reading on the impact of meta refresh redirects on SEO, refer to these resources:  

https://risify.app/meta-refresh-redirects/#negative-impact-of-meta-refresh-redirects-on-seo
https://seodity.com/blog/redirects-guidline/#html-redirects
https://help.ahrefs.com/en/articles/2433739-what-is-meta-refresh-redirect-and-why-is-it-considered-a-critical-issue
https://firozahmad.com/what-is-meta-refresh-redirection/

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Comment by Yndira-E** [comment]
I suggest that in the next iteration, we automate the generation of the redirects list instead of writing them directly in the file. This can be based on the approach described in this article: https://www.aleksandrhovhannisyan.com/blog/eleventy-netlify-redirects/

**Comment by sumitshinde-84** [comment]
> @sumitshinde-84 I think it’s working now. The errors don’t seem to be related to the `redirects` file but to the .md files that still have the old links.

Yes, it was working, but the href link tests are failing. It's related to the package because the current package isn't working well with server-side redirects.

File: src/redirects.njk [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+permalink: /_redirects
```
Comment (ZJvandeWeg) [inline src/redirects.njk:2]: I'm confused why this file exists, and the file without this extension?

Comment (sumitshinde-84) [reply src/redirects.njk:2]: It’s not ready yet; the review was requested automatically.

File: .eleventy.js [hunk 2]
```diff
@@ -85,6 +85,9 @@ module.exports = function(eleventyConfig) {
         "src/public/": "/",
     });
 
+    // Copy the contents of the `_redirects` to the output folder
+    eleventyConfig.addPassthroughCopy("src/_redirects");
```
Comment (ZJvandeWeg) [inline .eleventy.js:89]: Don't think this is needed if we keep the redirect.njk?

File: src/redirects.njk (hunk 1 above)
Comment (ZJvandeWeg) [reply src/redirects.njk:2]: I think it already works on my machine?

Comment (ZJvandeWeg) [reply src/redirects.njk:2]: Keep this file, remove the changes in .eleventy.js and it's ready?

File: .eleventy.js (hunk 2 above)
Comment (sumitshinde-84) [reply .eleventy.js]: yes, i have removed it

**Review by Yndira-E — APPROVED** [review]

**Pull request merged**